src/lucky_migrator/create_table_statement.cr

```diff
@@ -1,6 +1,9 @@
 class LuckyMigrator::CreateTableStatement
   getter statement = IO::Memory.new
   getter rows = [] of String
+  getter indices = [] of String
+
+  ALLOWED_INLINE_INDICES = %w[btree]
```

Comment:
Should we allow any other types? I think if we have this option it would be nice to allow more, but I'm also ok with just btree for now. I'm curious what your thoughts are on it